griffin-doc/service/api-guide.md

```diff
@@ -549,7 +549,21 @@ curl -k -H "Content-Type: application/json" -H "Accept: application/json" \
 `POST /api/v1/jobs/trigger/{job_id}`
 #### API Example
 ```
-curl -k -X POST http://127.0.0.1:8080/api/v1/jobs/trigger/51
+curl -k -X POST http://127.0.0.1:8080/api/v1/jobs/trigger/51 \
+-d '{
+    "timeout": "1000"
```

Comment:
what's the purpose of this timeout?

Reply:
This timeout is the maximum time in milliseconds that we except when creating JobInstanceBean.
The time between schedule job with "execute right now" trigger and the actual execution the job may be differ (from several milliseconds to tens of seconds). 
Sometimes, for example, in requests from the UI, waiting for a JobInstanceBean to be created is not required, because response will be ignored. 
But in other cases it can be very important. For example, we can run job using this API from ETL. After that using JobInstance ID from response, we can check the measurement result.

Reply:
Please make sure both cases (body is specified, and body is not specified) are covered with tests. Let wait by default (with default specified in config), and specify 0 explicitly, if required.